contrib/nydusify/pkg/viewer/viewer.go

```diff
@@ -61,11 +61,29 @@ func New(opt Opt) (*FsViewer, error) {
 	}
 
 	mode := "cached"
-	digestValidate := true
+	digestValidate := false
 	if opt.FsVersion == "6" {
 		mode = "direct"
 		digestValidate = false
 	}
+	// targetParsed, err := targetParser.Parse()
```

Comment:
Why add these useless codes here.

Reply:
This part has been solved in the new pull-request #1317